charmcraft/templates/init-simple/tox.ini.j2

```diff
@@ -2,27 +2,27 @@
 # See LICENSE file for licensing details.
 
 [tox]
-skipsdist=True
+no_package = True
 skip_missing_interpreters = True
-envlist = fmt, lint, unit
+env_list = format, lint, unit
```

Comment:
These updates will cause tox to fail if the user has tox <= 4.0. I the user installed tox through apt on any current version of Ubuntu that'll be the case, but you can have tox >=3.8.0 (available in apt on focal and later) handle that with a bootstrap venv:

```suggestion
env_list = format, lint, unit
minversion=4.0.0
requires = 
    tox>=4.0.0
```